### PR TITLE
`unread-anywhere` - Add feature

### PR DESCRIPTION
**Pull request by fregante**
- Closes https://github.com/refined-github/refined-github/issues/7442

## Test URLs

anywhere :)

## Screenshot

doubleclick

### CODE REVIEW

File: source/features/unread-anywhere.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const limit = 10;
+
+async function openUnreadNotifications(): Promise<void> {
+	await showToast(async updateToast => {
+		const page = await fetchDomUncached('https://github.com/notifications?query=is%3Aunread');
+
+		const notifications = $$optional('a.js-navigation-open', page);
+		if (notifications.length === 0) {
+			updateToast('No unread notifications');
```
Comment (fregante) [inline source/features/unread-anywhere.tsx:20]: fail

File: source/features/unread-anywhere.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		}
+
+		updateToast('Opening…');
+		const urls = notifications.slice(0, limit).map(notification => notification.href);
+		await messageRuntime({
+			openUrls: urls,
+		});
+
+		if (notifications.length > limit) {
+			updateToast(`Opened the last ${limit} unread notifications`);
```
Comment (fregante) [inline source/features/unread-anywhere.tsx:31]: When limited:

full

File: source/features/unread-anywhere.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		} else {
+			updateToast(pluralize(urls.length, '$$ notification') + ' opened');
+			// Update the UI too
+			$('.AppHeader-button--hasIndicator').classList.remove('AppHeader-button--hasIndicator');
```
Comment (fregante) [inline source/features/unread-anywhere.tsx:35]: Opening all the notifications, then removing the blue bubble:

doubleclick

**Comment by fregante** [comment]
This feature has been added as a shortcut for now. We can work it into the UI at a later stage. I just want it in the next release 🙂

File: source/features/avoid-accidental-submissions.tsx [hunk 4]
```diff
@@ -3,8 +3,8 @@ import {elementExists} from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 import delegate, {DelegateEvent} from 'delegate-it';
 
-import {isMac} from '../github-helpers/index.js';
 import features from '../feature-manager.js';
+import {modKey as moduleKey} from '../github-helpers/hotkey.js';
```
Comment (forivall) [inline source/features/avoid-accidental-submissions.tsx:7]: isn't `mod` short for `modifier`, not `module`?

Comment (fregante) [reply source/features/avoid-accidental-submissions.tsx:7]: The dangers of eslint autofix 😅 

Yes it's modifier

**Pull request merged**